gravity.sh

```diff
@@ -1016,6 +1017,7 @@ migrate_to_listsCache_dir() {
   local str="Migrating the list's cache directory to new location"
   echo -ne "  ${INFO} ${str}..."
   mkdir -p "${listsCacheDir}"
+  chown pihole:pihole "${listsCacheDir}"
```

Comment:
```suggestion
  mkdir -p "${listsCacheDir}" && chown pihole:pihole "${listsCacheDir}"
```

Addin in that `&&` so the `chown` is only called if the directory is created.